themes/default/content/blog/disabledefaultproviders/index.md

```diff
@@ -0,0 +1,280 @@
+---
+title: "Unlock Programmatic Control by Disabling Default Providers"
+date: 2022-03-18
+meta_desc: Introducing the new config parameter disable-default-providers.
+authors: [ ian-wahbe ]
+tags: [ features ]
+---
+
+As of 3.23.0, users can disable the default provider with Pulumi. So what does this mean for you? If you’ve been using Pulumi for a bit, you’ll have encountered provider resources, which are how we abstract the global state of a cloud provider. All resources have an associated provider. If no provider is supplied in the user’s code, a default provider is created to serve the resource. On the other hand, explicit providers defined by the user allow programmatic and dynamic control of how a resource deploys into a cloud. A Pulumi resource can be instructed to use an explicit provider by setting the provider resource option or by inheriting the provider from the resource's parent resource.
```

Comment:
```suggestion
As of 3.23.0, users can disable the default provider with Pulumi. So what does this mean for you? If you’ve been using Pulumi for a bit, you’ll have encountered [provider resources]({{< relref "/docs/intro/concepts/resources/providers" >}}), which are how we abstract the global state of a cloud provider. All resources have an associated provider. If no provider is supplied in the user’s code, a [default provider]({{< relref "/docs/intro/concepts/resources/providers#default-provider-configuration" >}}) is created to serve the resource. On the other hand, explicit providers defined by the user allow programmatic and dynamic control of how a resource deploys into a cloud. A Pulumi [resource]({{< relref "/docs/intro/concepts/resources" >}}) can be instructed to use an explicit provider by setting the [provider resource option]({{< relref "/docs/intro/concepts/resources/options/provider" >}}) or by inheriting the provider from the resource's [parent resource]({{< relref "/docs/intro/concepts/resources/options/parent" >}}).
```

Reply:
I don't know why GitHub isn't picking up what I changed, but I added links in from the google doc.